# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmStubs.cpp
# Commit: 5718a8d121f9
# Full Hash: 5718a8d121f95ce740a11ee443444cea990d6323
# Author: Dmitry Bezhetskov <dbezhetskov@igalia.com>
# Date: 2020-11-03 21:39:33
# Regressor Bug: 1639153
# File Overlap Count: 1
# Description:
#   Bug 1639153 - Part 7: Remove Frame::tls. r=lth
#   
#   We remove Frame::tls and fix prologue/epilogue. Runtime, Ion and Baseline are ready for this move already.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D83064
# ==============================================================================

diff -r 40031ad88dbd -r 5718a8d121f9 js/src/wasm/WasmStubs.cpp
--- a/js/src/wasm/WasmStubs.cpp	Tue Nov 03 10:28:10 2020 +0000
+++ b/js/src/wasm/WasmStubs.cpp	Tue Nov 03 10:32:55 2020 +0000
@@ -2461,7 +2461,7 @@
     MIRTypeVector coerceArgTypes;
     MOZ_ALWAYS_TRUE(coerceArgTypes.append(MIRType::Pointer));
     unsigned offsetToCoerceArgv =
-        AlignBytes(StackArgBytesForWasmABI(coerceArgTypes), sizeof(Value));
+        AlignBytes(StackArgBytesForNativeABI(coerceArgTypes), sizeof(Value));
     MOZ_ASSERT(nativeFramePushed >= offsetToCoerceArgv + sizeof(Value));
     AssertStackAlignment(masm, ABIStackAlignment);
 